packages/gestalt/src/Tabs.js

```diff
@@ -25,54 +52,68 @@ export default function Tabs({
     text: React.Node,
   |}>,
   wrap?: boolean,
-|}): React.Node {
-  const handleTabClick = (i: number, e: SyntheticMouseEvent<>) =>
-    onChange({ activeTabIndex: i, event: e });
+|};
 
+export default function Tabs({
+  activeTabIndex,
+  onChange,
+  size = 'md',
+  tabs,
+  wrap,
+}: Props): React.Node {
   return (
-    <div
-      className={classnames(
-        styles.Tabs,
-        wrap && layout.flexWrap,
-        size === 'md' ? layout.medium : layout.large
-      )}
+    <Box
+      alignItems="center"
+      display="flex"
+      marginEnd={-1}
+      marginStart={-1}
+      minHeight={size === 'lg' ? 48 : 40}
       role="tablist"
+      wrap={wrap}
+      dangerouslySetInlineStyle={{
+        __style: { userSelect: 'none' },
```

Comment:
@jackhsu978 although it looks the same as the original CSS, it is not. In our CSS files we autoprefix our CSS, with inline styles that's not the case.

Looking at https://caniuse.com/#feat=user-select-none we should probably add the `-ms` and `-webkit` prefix versions here as well.

![image](https://user-images.githubusercontent.com/127199/86025017-957d8480-b9e2-11ea-9f8d-b7aa32d3b1b0.png)

Reply:
That's really good point. I changed my mind about using inline style for this. Instead of passing this as inline style to the whole Tab component wrapper, I move it to a `<Link>` CSS class that's only enabled when the `role` prop is `"tab"`. I think this is better because (1) no need to worry about vendor prefixes, (2) there's a direct logical association between `user-select: none` and the Link whose role is a tab, and (3) it's cleaner because there's less logic needed in `<Tabs>` component.